src/server/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIR_SERVER_0.1.5 {
+MIR_SERVER_0.1.6 {
```

Comment:
I'm not sure this is what I was supposed to do

Reply:
https://mir-server.io/doc/dso_versioning_guide.html

Reply:
So... this is the correct way to change a symbol and break backwards compatibility?

Reply:
Part of it. The soname needs bumping too.

Reply:
Oh, what does it mean to bump the soname? Doc just says when to do it.

Reply:
1. bump MIRSERVER_ABI
2. tools/update_package_abis.sh

Maybe you can improve the docs?

Reply:
When you say MIRSERVER_ABI, you mean change `MIR_SERVER_0.1.5` to `MIR_SERVER_0.1.6` in src/server/symbols.map, or is MIRSERVER_ABI a different thing?

Reply:
src/server/CMakeLists.txt:166

Reply:
Ack, will get it passing the tests and then push version bump

Reply:
Hm. We would traditionally call this `MIR_SERVER_1.5.0`, as that's the version this would be released in.

Reply:
Let's be consistent with ourselves and use MIR_SERVER_1.6.0 as that's the version this would be released in.